src/cascadia/Remoting/Peasant.cpp

```diff
@@ -126,14 +127,17 @@ namespace winrt::Microsoft::Terminal::Remoting::implementation
     // - <none>
     // Return Value:
     // - <none>
-    void Peasant::Summon()
+    void Peasant::Summon(const Remoting::SummonWindowBehavior& summonBehavior)
     {
-        _SummonRequestedHandlers(*this, nullptr);
+        auto localCopy = winrt::make_self<implementation::SummonWindowBehavior>(summonBehavior);

```

Comment:
nit: you could probably just `winrt::make` here to get the projected type outright. But meh.